JS-Algorithms/challenges.js

```diff
@@ -42,6 +51,25 @@ const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
   // YOUR CODE HERE...
+
+  let COUNTRY_COPY = [...COUNTRY_NAMES];
+  const COUNTRY_NAMES_LENGHT = COUNTRY_COPY.length;
+
+  if (index === COUNTRY_NAMES_LENGHT) return [...COUNTRY_COPY];
+
+  const GetIndex = ({ OldIndex }) =>
+    OldIndex > COUNTRY_NAMES_LENGHT
+      ? OldIndex % COUNTRY_NAMES_LENGHT
+      : OldIndex;
```

Comment:
This function can live outside of circularArray since not needed to be define each time the circularArray is being called

Reply:
There's no need for destructuring here since it just adds complexity. Instead, pass OldIndex directly as a parameter.